deploy/vagrant/run_vb.sh

```diff
@@ -5,11 +5,11 @@ vagrant up --provider=virtualbox --no-provision
 vagrant provision
 
 if [[ "$?" == "0" ]]; then
-    HERE=`dirname $0`
-    pushd $HERE > /dev/null
-    master=`tail -n 1 $HERE/files/hosts | cut -d' ' -f1`
-    purple='\033[1;35m'
-    no_color='\033[0m'
-    echo -e ">>> ${purple}visit $master:19999 for Tachyon Web Console${no_color} <<<"
-    echo -e ">>> ${purple}visit $master:8080  for Spark   Web Console${no_color} <<<"
+ HERE=`dirname $0`
+ pushd $HERE > /dev/null
+ master=`tail -n 1 $HERE/files/hosts | cut -d' ' -f1`
+ purple='\033[1;35m'
+ no_color='\033[0m'
+ echo -e ">>> ${purple}visit $master:19999 for Tachyon Web Console${no_color} <<<"
+ echo -e ">>> ${purple}visit $master:8080  for Spark   Web Console${no_color} <<<"
```

Comment:
Its kind of weird to have this line if they don't deploy spark, but its not really related to this PR.